vite-project/main.js

```diff
@@ -0,0 +1,150 @@
+    secondCross.classList.add('cross')
+    firstCross.appendChild(secondCross)
+
+    const boxRight = document.createElement('div')
+    boxRight.classList.add('app__box--choice')
+    ticTacBox.appendChild(boxRight)
+
+    const circle = document.createElement('div')
+    circle.classList.add('circle')
+    boxRight.appendChild(circle)
+
+    return ticTacBox
+}
+
+const createTicTacArena = () => {
+    const ticTacArena = document.createElement('div')
+    ticTacArena.classList.add('app__arena')
+    app.appendChild(ticTacArena)
+
+    return ticTacArena
+}
+
+const ticTacBox = createTicTacChoice()
+const arena = createTicTacArena()
+
+let cubeArray = Array.from(new Array(9))
+
+cubeArray.forEach((object, index) => {
+    const newCube = document.createElement('div')
+    newCube.classList.add('cube')
+    arena.appendChild(newCube)
+
+    newCube.addEventListener('click', () => {
+
+        const choice = document.querySelector('.app__box--choice.choice-green').firstChild.classList.value
+
+        if (newCube.classList.contains('cube') && choice === 'circle' && !newCube.hasChildNodes()) {
+            const myChoice = document.createElement('div')
+            myChoice.classList.add('circle')
+            newCube.appendChild(myChoice)
+            arena.style.pointerEvents = 'none'
+            cubeArray[index] = 'O'
+        }
+        if (newCube.classList.contains('cube') && choice === 'cross' && !newCube.hasChildNodes()) {
+            const firstCross = document.createElement('div')
+            firstCross.classList.add('cross')
+            newCube.appendChild(firstCross)
+
+            const secondCross = document.createElement('div')
+            secondCross.classList.add('cross')
+            firstCross.appendChild(secondCross)
+            arena.style.pointerEvents = 'none'
+            cubeArray[index] = 'X'
+        }
+    })
+})
+
+cubeArray.fill('')
+
+const makeChoice = event => {
+
+    const choice = document.querySelector('.app__box--choice.choice-green')
+
+    if (choice) {
+        choice.classList.remove('choice-green')
+    }
+
+    if (event.target.classList.contains('app__box--choice')) {
+        event.target.classList.add('choice-green')
+    }
+
+    if (event.target.classList.contains('circle') || event.target.classList.contains('cross')) {
+        event.target.parentNode.classList.add('choice-green')
+    }
+}
+
+ticTacBox.addEventListener('click', makeChoice)
+
+const allCube = document.querySelectorAll('.cube')
+
+const play = () => {
+
+    ticTacBox.removeEventListener('click', makeChoice)
+
+    const choice = document.querySelector('.app__box--choice.choice-green').firstChild.classList.value
+
+    let indexes = cubeArray.map((field, index) => field ? null : index)
+    let filtered = indexes.filter(item => item !== null)
+    let randomNumber = Math.floor(Math.random() * filtered.length)
+    let randomNumberValue = filtered[randomNumber]
+
+    if (randomNumberValue === undefined) {
```

Comment:
`if(randomNumberValue)`

Reply:
if i do it, i will always have a alert 'draw', from this if